### PR TITLE
Login flow css

### PR DESCRIPTION
**Pull request by emman4github**
1. CSS has been fixed for both Signup and Login flow.
2. Prayer request top whitespace has been removed and fixed.
3. Enabled the Raise Prayer Request for non logged in/ signup user, if they try to click it will prompt an alert saying kindly login/signup.

### CODE REVIEW

File: nlfyapp/src/components/expandCollapse.CommunityPrayer.component.js [hunk 1]
```diff
@@ -211,22 +210,19 @@ export const ExpandCollapseListCommunityPrayer = (props) => {
     flatlistWrapper: {
       flex: 1,
       position: "absolute",
-      bottom: 0,
-      height: height * 0.64,
-      overflow: "scroll",
-      marginTop: width * 0.01,
+      height: height * 0.63,
+      //overflow: "scroll",
```
Comment (designgeeks777) [inline nlfyapp/src/components/expandCollapse.CommunityPrayer.component.js]: Please clean up and remove commented codes

File: nlfyapp/src/components/textinput.js [hunk 2]
```diff
@@ -1,3 +1,144 @@
+// import React, { useRef, useState } from "react";
+// import { View, Text, Dimensions } from "react-native";
+// import { TextInput } from "react-native-paper";
```
Comment (designgeeks777) [inline nlfyapp/src/components/textinput.js]: Whats this commented code?

File: nlfyapp/src/features/prayerRequest.screen.js [hunk 3]
```diff
@@ -19,13 +19,22 @@ const WrapperView = styled(View)`
   padding-top:${wrapperPadding }px;
 `;
 
+// const SafeAreaViewWrapper = styled(SafeAreaView)`
+//   flex: 1;
+//   padding-top: ${StatusBar.currentHeight * 0.5}px;
+//   // margin-top: ${StatusBar.currentHeight - 35 || 0}px;
```
Comment (designgeeks777) [inline nlfyapp/src/features/prayerRequest.screen.js]: CLear up commented codes

File: nlfyapp/src/components/expandCollapse.CommunityPrayer.component.js (hunk 1 above)
Comment (emman4github) [reply nlfyapp/src/components/expandCollapse.CommunityPrayer.component.js]: yaa i have cleaned up the code..

File: nlfyapp/src/components/textinput.js (hunk 2 above)
Comment (emman4github) [reply nlfyapp/src/components/textinput.js]: I have cleaned up the code

**Comment by emman4github** [comment]
i have cleaned up the code

**Comment by designgeeks777** [comment]
In textInput.js there are still some commented code. Can you re-check once?

**Review by designgeeks777 — APPROVED** [review]

**Pull request merged**